examples/default.yaml

```diff
@@ -20,7 +20,7 @@ vmOpts:
   qemu:
     # Minimum version of QEMU required to create an instance of this template.
     # Will be ignored if the vmType is not "qemu"
-    # 🟢 Builtin default: not set
+    # 🟢 Builtin default: not set (but RISC-V requires QEMU 9.1 or later)
     minimumVersion: null
```

Comment:
Maybe `minimumVersion` should be typed as `map[Arch] bool`? (in a separate PR)

Reply:
(Anyway riscv64 is experimental; I don't want to spend too much time for overengineering it)

Reply:
> I think it worked on earlier versions as well, as long as you supplied the firmware and kernel?

Yes, but s/and/or/

> Maybe the qemu implementation details can stay out of the default configuration comments.

Done